dmscripts/generate_framework_agreement_signature_pages.py

```diff
@@ -78,9 +81,19 @@ def render_html_for_successful_suppliers(rows, framework, template_dir, output_d
 
 
 def render_html_for_suppliers_awaiting_countersignature(rows, framework, template_dir, output_dir):
-    template_path = os.path.join(template_dir, 'framework-agreement-signature-page.html')
-    template_css_path = os.path.join(template_dir, 'framework-agreement-signature-page.css')
-    template_countersignature_path = os.path.join(template_dir, 'framework-agreement-countersignature.png')
+    output_dir = Path(output_dir).resolve()
+    html_pages = [Path(template_dir, 'framework-agreement-signature-page.html').resolve()]
+    static_files = [Path(template_dir, 'framework-agreement-signature-page.css').resolve()]
+
+    if not framework_supports_e_signature(framework['slug']):
+        static_files.append(Path(template_dir, 'framework-agreement-countersignature.png'))
+
+    if framework_supports_e_signature(framework['slug']):
```

Comment:
Nit: can we make this more obvious with `else`?

Reply:
good point, fixed in 60c936c